### ISSUE TITLE
TS6053 false positive when src has no scheme

### ISSUE THREAD

**Issue by breyed** [Microsoft/TypeScript#20986]
Consider the following reference to a script on a CDN with a local fallback. Its source URL has no scheme (causing the scheme matches that of the calling page, e.g. http or https).

``` html
<script src="//ajax.googleapis.com/ajax/libs/angularjs/1.5.5/angular.min.js"
        asp-fallback-src="~/lib/angular/angular.min.js"
        asp-fallback-test="window.angular">
</script>
```

There are two bugs:

Bug 1: Visual Studio IntelliSense reports warning TS6053: (JS) File '_WebsitePath_/Views/Shared/ajax.aspnetcdn.com/ajax/jquery/jquery-2.2.3.min.js' not found.

Bug 2: If you try to work around bug 1 by adding a scheme to the `src` attribute, the IntelliSense warning persists. You have to close the solution, delete the offending lines and reinsert them, or the like.

The second bug is not at all obvious. It took me a while before I realized I had worked around bug 1 but was still seeing the warning due to bug 2.

**Comment by mhegazy** [comment]
This seems to be an HTML editor issue. please file it under https://developercommunity.visualstudio.com  instead.

**Comment by breyed** [comment]
I moved the issue [here](https://developercommunity.visualstudio.com/content/problem/177154/ts6053-false-positive-when-src-has-no-scheme.html).

FWIW, the issue relates to TypeScript, not HTML. It sounds like a common editor handles both, though.
